Flutter/datagrid/column-types.md

```diff
@@ -556,6 +556,59 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid checkbox shape](images/column-types/flutter-datagrid-checkbox-shape.png)
 
+### Callbacks
+
+The `SfDataGrid` provides the following callback for checkbox interactions:
+
+* [onCheckboxValueChanged]() : This callback is triggered when the checkbox is selected or unselected (i.e., when the value of the checkbox changes).
```

Comment:
This callback is triggered when the checkbox is selected or deselected, either by tapping on the checkbox or by selecting or deselecting a row (i.e., when the value of the checkbox changes)